site/e2e/server/run_with_forward.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+workspace=${1:-}
+port=${2:-3000}
+
+if [[ -z "${workspace}" ]]; then
+  echo "Usage: $0 <workspace> [port]"
+  exit 1
+fi
+
+# Go to site.
+cd "$(dirname "$0")/../.."
```

Comment:
if the script breaks, will the user land in another dir?

Reply:
Nah, only if someone tries to source this script which is not an intended use-case.

Reply:
It would be cool to add a script-check to verify if it isn't directly called, but definitely not a blocker

Reply:
Tbh I think I should just rewrite this completely to remove the coder repo requirement locally. Feels like it’d just add confusion trying to document this. Essentially you could either run the script from the repo or download it and run it individually (curl | bash), we can then just pick the playwright version requirement directly from the workspace which mean it has a greater chance of working on custom branches too.